content/OTP/Guides/OTP_Walk-through.adoc

```diff
@@ -204,6 +206,7 @@ In your application, add the logic needed to process registration and authentica
 
 Enable users to register with your application or platform.
 
+
 *Step 1* Expose a connection to your UI that accepts the OTP for launching the application. For example, instruct your users to insert the YubiKey. See link:https://developers.yubico.com/OTP/[What is Yubico OTP?]. This connection sends the OTP download to the YubiCloud client for validation.
```

Comment:
Chris Halos: This connection sends the OTP download to the YubiCloud client for validation - huh?